study.md

```diff
@@ -44,21 +57,37 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
-```
+
+The views are the distinct sections of the website. I think a view state is the
+current configuration of all the elements in the UI that the user has manipulated.
+
+I'm not clear on what a front end router does exactly.
+
```

Comment:
View state is referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state. The front end router is what manages view-state by responding to user inputs, and directing them to the appropriate view!
Take a look at this repo!! https://github.com/ga-wdi-boston/spa-study/blob/master/study.md